holidayextras/ui-brand-hx
# Updaing the navbar for HX, moar variables

#### What does this PR do? (please provide any background)
- During project lounge i took it upon myself to tidy the tripapp styling. starting with the stuff at the very top.. the navbar.
- The current branding only really touched the basic styling of a navbar, and even includes stuff we have never and may never use.
- This PR tdying the the existing styles and add's real word branding and styling to existing but cleaned html.

#### What tests does this PR have?
None

#### How can this be tested?
- Yu can check this out in the Pattern Library
- Make sure you have `ui-brand-hx` locally and have switched to the branch `navbar`
- Navigate to the pattern library directory on your machine and checkout `navbar`
- `npm link ../ui-brand-hx` assuming pattern library and brand are in the same parent folder
- now `npm start` and look at navbar in organisms section.
- Test in both mobile and desktop

#### Screenshots / Screencast
<img width="1375" alt="screen shot 2016-01-18 at 17 09 46" src="https://cloud.githubusercontent.com/assets/778942/12397630/559c2452-be06-11e5-9b83-a6e444fc7e16.png">
<img width="428" alt="screen shot 2016-01-18 at 17 09 56" src="https://cloud.githubusercontent.com/assets/778942/12397633/57d68104-be06-11e5-83d3-91ab398ca747.png">


#### What gif best describes how you feel about this work?
![Flight of the Conhords](https://media.giphy.com/media/xTiTneUZ6NoHdYpeE0/giphy.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
